spec/firebase_app_distribution_api_client_spec.rb

```diff
@@ -2,9 +2,8 @@
   let(:fake_binary_path) { "binary_path" }
   let(:fake_binary_contents) { "Hello World" }
   let(:fake_binary) { double("Binary") }
-  let(:fake_auth_client) { double("auth_client") }
 
-  let(:api_client) { Fastlane::Client::FirebaseAppDistributionApiClient.new }
+  let(:api_client) { Fastlane::Client::FirebaseAppDistributionApiClient.new("") }
```

Comment:
Can you pass in an actual value here, and then add expectations to these tests to check that the authentication token is actually used? I think that was missing before.